Example/Backpack/View Controllers/NavigationBarViewController.swift

```diff
@@ -0,0 +1,83 @@
+//
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright © 2019 Skyscanner Ltd. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import UIKit
+import Backpack
+
+class NavigationBarViewController: UIViewController {
+    private static let CellIdentifier = "CellIdentifier"
```

Comment:
Should this identifier be more specific to avoid clashes?

Reply:
There's no chance of that since we only register the on cell here, but I can change the name